source/common/modules/markdown-editor/index.ts

```diff
@@ -190,65 +200,42 @@ export default class MarkdownEditor extends EventEmitter {
   constructor (
     anchorElement: Element|DocumentFragment|undefined,
     editorId: string,
```

Comment:
Should we rename this to `leafId` to not confuse it with `this.editorId`?

Reply:
Good catch. I also took the liberty to update the constructor docstring, since that has changed quite a bit :D